templates/Service/_logger/logger.twig

```diff
@@ -34,6 +34,8 @@ final class {{ class }} implements LoggerInterface {
    * {@inheritdoc}
    */
   public function log($level, string|\Stringable $message, array $context = []): void {
+    // Cast message to string
```

Comment:
Casting is a trivial operation. I think this comment is redundant.

Reply:
True